unstructured/chunking/title.py

```diff
@@ -20,6 +20,9 @@
     Text,
     Title,
 )
+from unstructured.utils import lazyproperty
+
+_Section: TypeAlias = Union["_NonTextSection", "_TableSection", "_TextSection"]
```

Comment:
Do we want to start using the 3.10 syntax with `|`?

Reply:
Hmm, I'm glad you mentioned this, turns out it actually can as long as the whole expression is surrounded in quotes.
I was having trouble finding a clear answer on search and this was the best I could come up with, but having another search now I turned up by accident that this will work:

```python
_Section: TypeAlias = "_NonTextSection | _TableSection | _TextSection"
```

I'll change this to that. The quotes are required because this actually gets parsed as an assignment statement and prior to 3.10 the `from __future__ import annotations` can't change that particular behavior.

Still, the stringified version is much more pleasing and also consistent of course. Thanks for mentioning this! :)